sdk/storage/inc/datalake/directory_client.hpp

```diff
@@ -65,6 +65,13 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
         const std::string& directoryUri,
         const DirectoryClientOptions& options = DirectoryClientOptions());
 
+    /**
+     * @brief Create a FileClient from current DirectoryClient
+     * @param path Path of the file under the directory.
+     * @return FileClient
+     */
+    FileClient GetFileClient(const std::string& path) const;
```

Comment:
Is there something like `GetSubdirectory` in other SDK? If we have function to get a file under a directory, maybe it's also reasonable to get a subdirectory.

Reply:
If we do have this function in other SDK, I think you dont have to do it now, creating an issue to track it is also fine.

Reply:
Yes, but Jeffrey did not bring this up during the API review so I don't think it's currently necessary to add. 
Created https://github.com/Azure/azure-sdk-for-cpp/issues/338 to track.